frontend/src/metabase/query_builder/utils.unit.spec.ts

```diff
@@ -307,6 +333,14 @@ describe("isNavigationAllowed", () => {
       ).toBe(true);
     });
 
+    it("allows to run the model and then edit it again", () => {
```

Comment:
This new test is in `when creating new model` block, but there's no new test in `when creating a new structured question` should we do the same?

I was wondering because there are 2 new tests for models, but only 1 for questions

Reply:
It's because #34686 is not implemented yet.